ckcp/openshift/base/ingress.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
```

Comment:
In Openshift cluster, using ` --dry-run=server` can't get hostname. We have to revert it back to Route CR.

Reply:
Clarified offline. Marking it resolved.